# dynamixel goal velocity
GOAL_VELOCITY: 10

# deformation
DEFORMATION_TH: 5.0

# deformation
OCCLUSION_TH: 4.0

# hsv param: hand
H_MIN_HAND: 98
H_MAX_HAND: 179
S_MIN_HAND: 0
S_MAX_HAND: 255
V_MIN_HAND: 125
V_MAX_HAND: 255

# hsv param: obj
H_MIN_OBJ: 0
H_MAX_OBJ: 100
S_MIN_OBJ: 100
S_MAX_OBJ: 255
V_MIN_OBJ: 100
V_MAX_OBJ: 255